website/src/pages/en/ai-suite/graph-assistant/quick-start.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Quick Start
+---
+
+This guide provides step-by-step instructions for getting started with Graph AI Assistant quickly.
+
+## Step-by Step
+
+### Step 1. Create an Account
+
+Go to: (Graph AI)[thegraph.com/ai]
+
+- Click Sign Up (or Log In if you already have an account).
+- Use your GitHub or email credentials.
+- No additional setup is required
+
+### Step 2. Open the Assistant Chat
+
+After logging in:
+
+- You'll see the assistant interface.
+- At the bottom of the page, you will find a chat input bar. This is where you will interact with the assistant.
+
+### Step 3: Ask Your First Question
+
+> [!IMPORTANT] All users can process 10 free queries per user per hour
+
+In the input bar, type a natural-language query (everyday language).
+
+Example:
+
+```yaml
+Show me the 24-hour trading volume of XYZ token on Ethereum
+```
+
+The assistant will:
+
+- Digest your question
+- Fetch onchain data using Subgraphs or the Token API
+- Return a clean, formatted response in markdown with the data you need
```

Comment:
They don't need to know it's a markdown table. It'll just show up a table on their side.